dictionaries/r/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017-2020 Matthew Toohey <contact@mtoohey.com>
```

Comment:
You can add your name, but please keep mine as well, otherwise it can make publishing difficult as the dictionary evolves over time.

Reply:
Sure, resolved in 1e1e622d74176624f0d4eda937f444676c231cb9.